<?xml version="1.0" encoding="UTF-8" ?>
<testsuite errors="0" failures="1" hostname="laptop-PC" name="com.xaviar.service.UserServiceTests" tests="1" time="0.329" timestamp="2013-08-05T04:19:49">
  <properties />
  <testcase classname="com.xaviar.service.UserServiceTests" name="testSomething" time="0.06">
    <failure message="junit.framework.AssertionFailedError: Implement me" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: junit.framework.AssertionFailedError: Implement me
	at com.xaviar.service.UserServiceTests.testSomething(UserServiceTests.groovy:15)
</failure>
  </testcase>
  <system-out><![CDATA[--Output from testSomething--
]]></system-out>
  <system-err><![CDATA[--Output from testSomething--
]]></system-err>
</testsuite>
